src/include/meson.build

```diff
@@ -0,0 +1,95 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+include_sources = files(
+	'bpf.h',
+	'bpf_common.h',
+	'netns_ifaddrs.c',
+	'netns_ifaddrs.h')
+
+if cc.has_function('getline', prefix : '#include <stdio.h>', args : '-D_GNU_SOURCE')
+	conf.set10('HAVE_GETLINE', have)
+else
+	include_sources += files(
+		'getline.c',
+		'getline.h')
+endif
+
+if cc.has_function('fexecve', prefix : '#include <unistd.h>', args : '-D_GNU_SOURCE')
+	conf.set10('HAVE_FEXECVE', have)
```

Comment:
Lots of leaky reuse of the "have" variable here...

Reply:
removed